redaxo/src/core/lib/Fragment/Component/Button/Save.php

```diff
@@ -11,7 +11,7 @@
 use Redaxo\Core\Fragment\HtmlAttributes;
 use rex_i18n;
 
-class Save extends Fragment
+final class Save extends Fragment
```

Comment:
In den Button-Subklassen machen wir das bereits auch so. Also `Button\Save` erbt nicht von `Button`, sondern verwendet `Button` nur intern.